docs/pages/connect-your-client/tsh.mdx

```diff
@@ -842,5 +842,59 @@ DEBU [TSH]       Self re-exec command: tsh [status --format=json]. tsh/aliases.g
 ...
 ```
 
+# Interactive Recording
+
+Within the Teleport cluster the `ssh` and `Kubernetes` recorded sessions are available for listing and replay for authorized users via `tsh`.
+The same listings including Desktops are available in the Web Console under the Audit Log in the Activity section.
+
+## List Recordings and Play
+
+Get the list of recordings.  Without specifying a time period the past 24 hour recordings are shown.
+```code
+$ tsh recordings ls
+ID                                   Type Participants Hostname Timestamp           
+------------------------------------ ---- ------------ -------- ------------------- 
+b0a04442-70dc-4be8-9308-7b7901d2d600      jeff         dev       Nov 26 16:36:16 UTC 
+c0a02222-70dc-4be8-9308-7b7901d2d600 kube alice                  Nov 26 20:36:16 UTC 
+d0a04442-70dc-4be8-9308-7b7901d2d600      navin        test      Nov 26 16:36:16 UTC 
+```
+
+If no type is specified that is a `ssh` session.
```

Comment:
This seems like an oversight. We should fix this in the code.

Reply:
Addressing this in another PR